cms/apps/media/admin.py

```diff
@@ -107,6 +108,11 @@ class FileAdmin(VersionAdmin, SearchAdmin):
     search_fields = ['title']
     ordering = ['pk']
 
+    def get_form(self, request, obj):
+        if obj.is_image:
+            return ImageChangeForm
```

Comment:
This would be better, and matches the super's `get_form` method signature:

```
    def get_form(self, request, obj=None, **kwargs):
        if obj.is_image:
            kwargs['form'] = ImageChangeForm
        return super().get_form(request, obj, **kwargs)
```

Reply:
Also, `obj` can be None. I'd expect this to break adding a new file.